### ISSUE TITLE
Underline inline style outputs code on the front end

### ISSUE THREAD

**Issue by kathrynwp** [Automattic/gutenberg-everywhere#140]
While the Underline option is provided as an inline style, it isn't displayed correctly.

### To replicate

- Select some text in a forum post
- Click the upside-down caret icon and apply the Underline option
- Submit the post

### Result

Before submission the underlined text looks OK in the editor:

<img width="363" alt="A____Would_Review_Again_-__Test_Plugin_3__Review___WordPress_org" src="https://user-images.githubusercontent.com/7662449/215850771-57752c2a-05ed-40ef-ac94-422ccdd912cf.png">

Once submitted, inline HTML is visible on the front end:

<img width="549" alt="A____Would_Review_Again_-__Test_Plugin_3__Review___WordPress_org" src="https://user-images.githubusercontent.com/7662449/215850990-49fe1251-f8de-4308-8f31-a6331cdc7b11.png">

### Expected

Underlining should be displayed on the front end instead of HTML and inline CSS.

### Note

The inline options appear to be different in the live WP.org Support forums than in the [test forum](https://test.wordpress.org/support/forum/test/), which may be why this was not caught earlier. If it would be possible to get the test forum in sync with the live forum, it would be helpful.

**Test Forum Inline Options**

<img width="520" alt="Test___WordPress_org_Testing_Rosetta" src="https://user-images.githubusercontent.com/7662449/215849603-7db9aae0-1a0c-4ad8-a12f-252a0cd43a75.png">

**Live Forum Inline Options**

Includes Justify & Underline options:

<img width="435" alt="Localhost_Installs___WordPress_org" src="https://user-images.githubusercontent.com/7662449/215849836-51f7f916-06f1-4bbb-b8f4-c49bf31bb7a2.png">

### Report

[Reported in Slack here](https://wordpress.slack.com/archives/C02RQC6RW/p1675163824728039)

**Comment by johngodley** [comment]
Where are you seeing the underline option? It is not part of the plugin.

**Comment by kathrynwp** [comment]
@johngodley Weird! It's in the live forums, for example when creating a post:

https://wordpress.org/support/forum/how-to-and-troubleshooting/?view=all#new-topic-0

**Comment by johngodley** [comment]
It's not part of Gutenberg itself, or this plugin, so must be something added on that forum but not the test ones. If it is supposed to be there then it will need to be added to the KSES filters.

**Comment by kathrynwp** [comment]
OK! What's the best way to address this, do you think I should make a Meta ticket, or something else?

**Comment by kathrynwp** [comment]
I don't think we need underlining or justification in the forums, personally.

**Comment by porg** [comment]
Underlining may be important when users want to mimick a link without creating a real one, when paraphrasing an error message, bug or proposal. A situation which is realistic on a support forum for a CMS.

**Comment by porg** [comment]
Or when quoting text which has underlines, which is possible in Gutenberg.

Why limit expression?

**Comment by johngodley** [comment]
@kathrynwp I don't know what the process is, maybe @dd32 can advise. If it's not intentional then it's probably a plugin or configuration thats running on the main forums but not the test one.

**Comment by johngodley** [comment]
Actually, trying the wp.com editor it appears there (along with some others) so I'm guessing it's Jetpack.

**Comment by porg** [comment]
- Could that also be the root cause for #135 ?

**Comment by johngodley** [comment]
> Could that also be the root cause for https://github.com/Automattic/blocks-everywhere/issues/135

No. As far as I know that is now fixed.
